sql/moz-fx-data-shared-prod/fenix/marketing_attributable_metrics/view.sql

```diff
@@ -6,5 +6,7 @@ SELECT
   1 AS dau,
   IF(is_new_profile, 1, 0) AS new_profiles,
   searches AS search_count,
+  IF(is_new_install, 1, 0) AS new_installs,
```

Comment:
This is purely for my edification; what distinguishes a _new_ install? Is this, like, a person who has never installed before and is _not_ reinstalling to update?

Reply:
I just added this, but this is basically to account for the [missing Ad clicks & CDOU incident](https://docs.google.com/document/d/1r1RcRstytYN21sxl82Cbr4ueGErLe6Q7ho_hsQxrg2o/edit#). We have a pretty good sense of when an ID is regenerated vs. a new install, so I'm counting _just_ new installs here (or approximating them, at least).